### PR TITLE
style: update harness to eliminate I2041 flake8 errors

### PR DESCRIPTION
**Pull request by MikhailKardash**
## Description

Update a lot of imports and variables to solve I2041 flake8 issues.

## Test Plan

No testing required

## Checklist

- [ ] Changes have been manually QA'd
- [ ] User-facing API changes need the "User-facing API Change" label.
- [ ] Release notes should be added as a separate file under `docs/release-notes/`.
  See [Release Note](https://github.com/determined-ai/determined/blob/master/docs/release-notes/README.md) for details.
- [ ] Licenses should be included for new code which was copied and/or modified from any external code.

## Ticket
MD-13

### CODE REVIEW

File: harness/determined/lightning/experimental.py [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# type: ignore
+#  type: ignore
```
Comment (ioga) [inline harness/determined/lightning/experimental.py:1]: is this intentional?

File: harness/determined/profiler.py [hunk 2]
```diff
@@ -957,18 +957,18 @@ def make_labels(
...
-    def convert_to_timestamp_str(timestamp: datetime) -> str:
+    def convert_to_timestamp_str(timestamp: datetime.datetime) -> str:
         """
-        Convert a datetime object to the string format expected by the API. All timestamps must be
-        timezone-aware datetime.datetimes in UTC.
+        Convert a datetime.datetime object to the string format expected by the API. All timestamps
+        must be datetime.timezone-aware datetime.datetime.datetime.datetimes in UTC.
         """
-        # https://docs.python.org/3/library/datetime.html#determining-if-an-object-is-aware-or-naive
+        # https://docs.python.org/3/library/datetime.datetime.html#determining-if-an-object-is-aware-or-naive
```
Comment (ioga) [inline harness/determined/profiler.py:965]: revert the changes in the docstring and the URL?

File: harness/tests/experiment/keras/test_keras_data.py [hunk 3]
```diff
@@ -1,17 +1,17 @@
-# type: ignore
+#  type: ignore
```
Comment (ioga) [inline harness/tests/experiment/keras/test_keras_data.py:1]: also curious why this has an added space, ...

File: harness/tests/experiment/keras/test_tf_keras_trial.py [hunk 4]
```diff
@@ -1,6 +1,6 @@
 # type: ignore
```
Comment (ioga) [inline harness/tests/experiment/keras/test_tf_keras_trial.py:1]: ... but this line is unchanged

**Review by ioga — APPROVED** [review]

File: harness/determined/lightning/experimental.py (hunk 1 above)
Comment (MikhailKardash) [reply harness/determined/lightning/experimental.py:1]: No. I tried to remove it at first, but when I put it back in I mistyped. :|

File: harness/determined/cli/_util.py [hunk 5]
```diff
@@ -5,27 +5,30 @@
 import termcolor
 
 from determined import cli
+
+# avoid circular import
+from determined.cli import _declarative_argparse as detparse
```
[the remark this answers is not in the document]
Comment (MikhailKardash) [reply harness/determined/cli/_util.py:10]: Yeah, it's a bit awkward. Another solution would be to refactor parts of this file into `_declarative_argparse.py`.

**Review by azhou-determined — APPROVED** [review]
👏 this is great
